src/edit/Toolbar/PosterFrameDialog/index.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <tailor-dialog
+    v-model="dialog"
+    @click:outside="reset"
+    header-icon="mdi-image-multiple"
+    width="608">
+    <template #activator="{ on, attrs }">
+      <select-poster-btn v-on="on" v-bind="attrs" :is-disabled="isDisabled" />
+    </template>
+    <template #header>
+      Poster frame
+    </template>
+    <template #body>
+      <p class="mb-3 text-left">
+        Set an image that's displayed before the video is played.
+      </p>
+      <v-img :src="currentPosterFrame" class="frame" />
+      <select-poster
+        @select="selectedIndex = $event"
+        :generated-poster-frames="generatedPosterFrames"
```

Comment:
We could simplify/generalize `SelectPoster` prop name:
```suggestion
        :options="generatedPosterFrames"
```